tests/webidl/post.js

```diff
@@ -140,6 +140,14 @@ console.log('getAsArray: ' + new TheModule.Inner().getAsArray(12));
 new TheModule.Inner().mul(2);
 new TheModule.Inner().incInPlace(new TheModule.Inner());
 
+// extend a namespaced class from JS
+var iu = new TheModule.InnerUser();
+
+iu.Callback = function (inner) {
+	console.log('*js virtual namespaced replacement*');
+};
+iu.Callback(new TheModule.Inner());
+
```

Comment:
I'm unsure what this tests. It does
```
x.y = function(z) { console.log('string') };
x.y();
```
That seems like it would work before this PR, wouldn't it? Even if `y` didn't exist as a property, in JS it's being added, and also it ignores the parameter being passed in. Or am I missing something?

Reply:
Its actually the compilation of the test rather than running it that addresses the issue I saw. Before the PR the test wouldn't compile because the definition of Callback() in test.h doesn't match the definition Callback() in test.idl, the pointer vs value issue.

I added this logging into the function just for completeness to show that you could instantiate, override and call the function correctly with the changes. I'm happy to get rid of it if you feel its unnecessary.

Reply:
I see, thanks. Yeah, if it's a compilation fix then let's remove the runtime testing parts, they may be confusing.

Reply:
Sorted, I also added a comment to explain that InnerUserBase is a compilation check.